tables/es-g1.ctb

```diff
@@ -32,3 +32,144 @@ include litdigits6Dots.uti
 numsign 3456
 capsletter 46
 begcapsword 46-46
+
+# ----------------------------------------------------------------------------------------------
+
+# Emphasis
+emphclass italic
+emphclass underline
+emphclass bold
+emphclass script
+
+begemph bold 35
```

Comment:
@ultraleetj What do you think about these emphasis indicators? Do they make sense?